### DISCUSSION TITLE
open metaverse:  year in review

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#491] [General]
Please add highlights, developments, noteworthy news from the past year relating to the open metaverse.

**Comment by indiebio** [comment]
Hubs - crisis, then community rallying

**Comment by indiebio** [comment]
glTF progress: https://omigroup.github.io/gltf-extensions/ @omi-gltf-subgroup deployed mkdocs site version of the github repo with search integrated

**Comment by indiebio** [comment]
IPSME and OMI mentioned in academic article: 
"the IPSME architecture by Kim Nevelsteen (OMI member) and Martin Wehlou (https://doi.org/10.1145/3458307.3460966) stands out for its integration of disparate systems. It uses a publish-subscribe
mechanism and dynamic translators to enable interoperability without requiring uniform protocols, thereby supporting system
evolution and simplifying integration." (Yang et al 2024 - https://arxiv.org/html/2403.05205v1)

**Comment by indiebio** [comment]
Hardware: Apple Vision Pro launched

**Comment by indiebio** [comment]
Advances in AI

**Reply by indiebio** [reply]
Skyboxlabs full immersive terrain
full simulations...

**Comment by KEYFRAME-SEHU** [comment]
Creating Gen AI inside a Metaverse Platform https://www.youtube.com/watch?v=41JB0eE7XQU

**Comment by indiebio** [comment]
Local storage, and the W3C group formation:

**Comment by indiebio** [comment]
OMI is still healthy and strong! 
Facebook changed their name to Meta in October 2021. Something else of significance is Meta's 180 on their stance of open vs. closed. They were essentially pushing for a super closed metaverse and tried to own the name, while their AI and generative works are taking more of an OMI ethos

**Reply by indiebio** [reply]
https://www.businessinsider.com/mark-zuckerberg-open-source-ai-platforms-future-competition-apple-llama-2024-9
